devel/simple-websocket-server/Portfile

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4
+
+PortSystem          1.0
+PortGroup cmake     1.0
```

Comment:
The same comments on simple-web-server apply to this Portfile.

Reply:
fixed 'm both